swagger.yaml

```diff
@@ -0,0 +1,129 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Tech Holding Classes
+  description: An app to manage Tutorials
+host: localhost:8000
+basePath: "/"
+tags:
+- name: tutorials
+  description: API for user in the system
+schemes:
+- http
+consumes:
+- application/json
+produces:
+- application/json
+paths:
+  tutorial:
+    parameters:
+    - name: id
+      in: path
+      required: true
+      description: ID of tutorial that we want to find
+      type: integer
+    get:
+      tags:
+      - tutorials
+      summary: Get all tutorials in system
+      responses:
+        '200':
+          description: OK
+          schema:
+            "$ref": "#/definitions/tutorials"
+    post:
+      tags:
+      - tutorials
+      description: Create new tutorial in system
+      parameters:
+      - name: tutorial
+        in: body
+        description: User that we want to create
+        schema:
+          "$ref": "#/definitions/tutorial"
+      produces:
+      - application/json
+      responses:
+        '200':
```

Comment:
@jaimin45 @JainishaParmar  No need to wrap string in quotes if it doesn't have spaces.

CC: @jaimin45